src/tests/Microsoft.PowerFx.Core.Tests/TypeSystemTests/DTypeTests.cs

```diff
@@ -205,7 +208,36 @@ public void UnknownIsSubtypeOfAll()
             Assert.True(MultiSelectOptionSetType.Accepts(DType.Unknown));
             Assert.True(DType.Polymorphic.Accepts(DType.Unknown));
         }
-        
+
+        [Fact]
+        public void DeferredIsSubtypeOfAll()
```

Comment:
>DeferredIsSubtypeOfAll [](http://example.com/codeflow?start=20&length=22)

Could we have a DType[] of all types and use a for-loop here?

This may also help with the above functions.